rel-eng/openshift-sdn-node.service

```diff
@@ -1,8 +1,13 @@
 [Unit]
 Description=OpenShift SDN Node
-Before=openshift-node.service
 After=openvswitch.service
+After=openshift-sdn-master.service
+After=openshift-master.service
+Wants=openshit-sdn-master.service
+Wants=openshift-master.service
```

Comment:
It confuses me at least, so just the immediate dependencies that solve the ordering problem would be nice.
@mrunalp your opinion?